lib/initializers/scheduler.ls

```diff
@@ -8,9 +8,5 @@ module.exports = (robot) !->
 
   if process.env.DEBUG?
     schedule '*/10 * * * * *', \check-for-upcoming-events
-    # schedule '*/10 * * * * *', \prompt-standup
-    # schedule '*/13 * * * * *', \remind-about-standup
   else
     schedule '0 0 9 * * *', \check-for-upcoming-events
-    # schedule '0 0 8 * * *', \prompt-standup
-    # schedule '0 0 12 * * *', \remind-about-standup
```

Comment:
Do you feel like taking on converting this file to plain JavaScript?

Reply:
I've got my eyes on #37 in the near-future, but I didn't want to convert anything just yet.

Reply:
👍